flink-connector-obkv-hbase/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 OceanBase
+# flink-connector-oceanbase is licensed under Mulan PSL v2.
+# You can use this software according to the terms and conditions of the Mulan PSL v2.
+# You may obtain a copy of Mulan PSL v2 at:
+#         http://license.coscl.org.cn/MulanPSL2
+# THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+# EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+# MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+# See the Mulan PSL v2 for more details.
+
+com.oceanbase.connector.flink.OBKVHBaseDynamicTableSinkFactory
```

Comment:
What is the purpose of this file?

Reply:
This file is used to register it as one of DynamicTableSinkFactory implementation in Flink.